### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 17

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#193] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253736-7ae6e8ac-6893-4212-b975-b259a6727937.png)

**Comment by AxnNxs** [comment]
La dimostrazione è immediata se utilizziamo il binomio di Newton e lo applichiamo su 3 elevato alla n, che notiamo essere (1+2)^n. Tale sommatoria risultante coincide esattamente con il membro a destra dell’uguaglianza.

(1+2)^n = Sommatoria (con k =0, n) di (n scelgo k) * a^(n-k) * b^k = Sommatoria(1^(n-k) * 2^k * (n scelgo k))

(L’asterisco indica la moltiplicazione).
